radixdlt/src/main/java/com/radixdlt/CerberusModule.java

```diff
@@ -71,15 +78,15 @@ private PacemakerImpl pacemaker() {
 	private VertexStore getVertexStore(
 		Universe universe,
 		RadixEngine radixEngine,
-		@Named("self") EUID self
+		@Named("self") RadixAddress self

```

Comment:
What's the reasoning for using RadixAddress? RadixAddress is more of an application layer abstraction

Reply:
I recall there being an implementation reason for this, but it no longer applies, so I've reverted this.